### DISCUSSION TITLE
Suggestions by KingTino104

### DISCUSSION THREAD

**Discussion by KingTino104** [OvernodeProjets/Fixed-Heliactyl#9] [Ideas]
I think this will probably be hard to implement as you have not completely made Heliactyl but its possible,
i think you should add this as some hosts may like having email/username authentication instead of just using Discord.

**Comment by achul123** [comment]
Hi this is planned in my heliactyl fixed dev but it's not a priority in my list but it should happen in some time, if you have other suggestions don't hesitate to tell me.

**Comment by KingTino104** [comment]
You should probably add a option in the config to use a url for an iframe as ads. Instead of people having to implement it manually.
You should be able to use ejs and just put it somewhere where it fits instead of people having to do it manually, as manually means going through every single page hand by hand. For example there can be a ad below the login page form thingy.
It should be very easy as you should just make 2 more things in the settings.json e.g.
ads:
 enabled: true/false
 url: the url of the ads

```
<% if (settings.ads.enabled == true) { %>
<iframe data-aa="2311418" src="<%= settings.ads.url %>" style="width:468px; height:60px; border:0px; padding:0; overflow:hidden; background-color: transparent;"></iframe>
<% } %>
```

**Comment by KingTino104** [comment]
Also probably delete the server name option when editing the server or just make it change it on panel too

**Comment by KingTino104** [comment]
Heres another suggestion  - add a feature so going to for example /panellogin automatically logs you into the panel,
i've seen a few hosts (with a private dashboard) do this so it should be possible.

**Comment by achul123** [comment]
Yeah, but they have to use a personalized pterodactyl that means

**Comment by KingTino104** [comment]
> Yeah, but they have to use a personalized pterodactyl that means

Nope, i dont think so, since its just a post request to login it should be possible to just click a button and send the post request.

**Comment by KingTino104** [comment]
Add blacklisted users

**Comment by achul123** [comment]
i will check to add that

**Comment by achul123** [comment]
soon
![image](https://github.com/OvernodeProjets/Fixed-Heliactyl/assets/73477238/3a37f296-c22d-4cf5-b44d-f6fd05f8888b)

**Comment by achul123** [comment]
added

**Comment by KingTino104** [comment]
also i'd recommend that u look into the backend for vulnerabilites a bit more as there are many known vulnerabilities with heliactyl

**Comment by KingTino104** [comment]
use yaml instead of json

**Reply by KingTino104** [reply]
done in #14
